_data/redirects.yaml

```diff
@@ -4,11 +4,11 @@
 
 # sponsor
 - source: /sponsor
-  destination: /CSS_Sponsorship_Proposal.pdf
```

Comment:
Maybe add a redirect for this as well

Reply:
You mean if someone tries to navigate directly to cssbham.com/CSS_Sponsorship_Proposal.pdf ?

Reply:
yes

Reply:
yeah good point tbf, will do